actions/executions_resume.py

```diff
@@ -0,0 +1,25 @@
+from lib.action import St2BaseAction
+
+__all__ = [
+    'St2ExecutionsResume'
+]
+
+
+def format_result(item):
```

Comment:
this function looks identical to the one in the file above, it should go into the base action so code is not copy/pasted.

Reply:
:) agreed